### PR TITLE
272 frontend create alerts component

### PR DESCRIPTION
**Pull request by Kartik-M24**
Implemented functionality for exit button and added changes so alerts works for different screen sizes.

### CODE REVIEW

**Comment by Kartik-M24** [comment]
image
image

If you hover over X button it will expand and opacity decreases
image
image

**Comment by Kartik-M24** [comment]
I made some changes to the tailwind.config file is that algd? (added screen sizes)

File: client/tailwind.config.ts [hunk 1]
```diff
@@ -87,6 +87,15 @@ export default {
             fontWeight: "400"
           }
         ]
+      },
+      screens: {
+        xxs: [{ min: "320px" }],
```
Comment (bcho892) [inline client/tailwind.config.ts:92]: :+1: will be helpful in future

File: client/src/components/generic/AlertsComponent/AlertsComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+const AlertsComponent = ({ variant, message, isButton }: AlertsInputProp) => {
+  const [modalOpen, setModalOpen] = useState<boolean>(true)
+
+  const SuccessAlert = () => {
+    return (
+      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-[#109D27]">
+        <div className="ml-10 mr-5 h-5 w-5">
+          <TickIcon className="fill-white" />
+        </div>
+        <span className="xs:text-sm xs:text-center xxs:text-xs xxs:text-center left-20 px-1 uppercase text-white md:absolute md:ml-10">
```
Comment (bcho892) [inline client/src/components/generic/AlertsComponent/AlertsComponent.tsx:22]: rly nice job on these

File: client/src/components/generic/AlertsComponent/AlertsComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  variant?: "success" | "notification" | "error"
+  isButton?: boolean
+}
+
+const AlertsComponent = ({ variant, message, isButton }: AlertsInputProp) => {
+  const [modalOpen, setModalOpen] = useState<boolean>(true)
+
+  const SuccessAlert = () => {
+    return (
+      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-[#109D27]">
```
Comment (bcho892) [inline client/src/components/generic/AlertsComponent/AlertsComponent.tsx:18]: nit:
```suggestion
      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-green">
```

File: client/src/components/generic/AlertsComponent/AlertsComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            <XIcon className="fill-white" />
+          </button>
+        )}
+      </div>
+    )
+  }
+
+  const ErrorAlert = () => {
+    return (
+      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-[#9A141D]">
```
Comment (bcho892) [inline client/src/components/generic/AlertsComponent/AlertsComponent.tsx:42]: nit:
```suggestion
      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-red">
```

File: client/src/components/generic/AlertsComponent/AlertsComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            <XIcon className="fill-white" />
+          </button>
+        )}
+      </div>
+    )
+  }
+
+  const NotificationAlert = () => {
+    return (
+      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-[#FF6D04]">
```
Comment (bcho892) [inline client/src/components/generic/AlertsComponent/AlertsComponent.tsx:66]: nit:
```suggestion
      <div className="flex h-12 w-full items-center justify-between rounded-lg bg-orange">
```

**Review by bcho892 — APPROVED** [review]
nit: use the defined red, green and orange variables in `tailwind.config.ts` in case they change

File: client/tailwind.config.ts (hunk 1 above)
Comment (bcho892) [reply client/tailwind.config.ts:92]: normally I think tailwind has them as default, but in case we need to change in future this is fine

**Comment by bcho892** [comment]
also recommend you link the issue under the "developement" tab on the right

File: client/tailwind.config.ts (hunk 1 above)
Comment (Kartik-M24) [reply client/tailwind.config.ts:92]: Yup I kept the rest as default values but just added them in case later on.

**Pull request merged**